### PR TITLE
Add glossaries required for the `osu!catch Ranking Criteria`

### PR DESCRIPTION
**Pull request by Darius-Wattimena**
2nd try as my old PR didn't rebase correctly

A PR that will add links to the catch RC will follow

### CODE REVIEW

File: wiki/Glossary/Antiflow/en.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Antiflow
```
Comment (TicClick) [inline wiki/Glossary/Antiflow/en.md:5]: I think this, along with `Snapping`, also covers osu! as a mode, so I would place that under `/wiki/Disambiguation/Antiflow` (I wish we also had `/wiki/Flow`!)

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Glossary/Edge_dash/en.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Edge Dash
+
+An *edge dash* (also known as a *pixel dash*) is the term used in [osu!catch](/wiki/Game_mode/osu!catch) for a very large spacing between two [active objects](/wiki/Glossary/Active_object) where the required [trigger distance](/wiki/Glossary/Trigger_distance) is not reached, and as such, a [hyperfruit](/wiki/Hit_object/Hyperfruit) is not generated to create a [hyperdash](/wiki/Glossary/Hyperdash). The first object must be caught with the edge of the plate in order to catch the second object at all.
```
Comment (TPGPL) [inline wiki/Glossary/Edge_dash/en.md:7]:
```suggestion
The **edge dash** (also known as a *pixel dash*) is a term used in [osu!catch](/wiki/Game_mode/osu!catch) for a very large spacing between two [active objects](/wiki/Glossary/Active_object) where the required [trigger distance](/wiki/Glossary/Trigger_distance) is not reached, and as such, a [hyperfruit](/wiki/Hit_object/Hyperfruit) is not generated to create a [hyperdash](/wiki/Glossary/Hyperdash). The first object must be caught with the edge of the plate in order to catch the second object at all.
```

File: wiki/Glossary/Edge_dash/en.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Edge Dash
```
Comment (TPGPL) [inline wiki/Glossary/Edge_dash/en.md:5]:
```suggestion
# Edge dash
```

File: wiki/Glossary/Hyperdash/en.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Hyperdash
+
+A *hyperdash* is the term used in [osu!catch](/wiki/Game_mode/osu!catch) for the movement which is done when a [hyperfruit](/wiki/Hit_object/Hyperfruit) is generated. When modding a beatmap people tend to link the hyperfruit and the target [active object](/wiki/Glossary/Active_object) to define a hyperdash movement (e.g. `00:01:000 (1,2) - ` where 1 is the hyperfruit and 2 the target active object).
```
Comment (TPGPL) [inline wiki/Glossary/Hyperdash/en.md:7]:
```suggestion
The **hyperdash** is a term used in [osu!catch](/wiki/Game_mode/osu!catch) for the movement which is done when a [hyperfruit](/wiki/Hit_object/Hyperfruit) is generated. While modding beatmaps, people tend to link the hyperfruit and the target [active object](/wiki/Glossary/Active_object) to define a hyperdash movement (e.g. `00:01:000 (1,2) -`  where 1 is the hyperfruit, and 2 the target active object).
```

File: wiki/Glossary/Snapping/en.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Snapping
+
+In [osu!catch](/wiki/Game_mode/osu!catch) every [dash](/wiki/Glossary/Dash) and [hyperdash](/wiki/Glossary/Hyperdash) can be clasified into two different snappings, basic-snapped or higher-snapped.
+
+## Basic-snapped
+
+The term *basic-snapped* is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) whose time between [active objects](/wiki/Glossary/Active_object) is at least twice the time required to allow dashes or hyperdashes, respectively. As an example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic-snapped hyperdash.
```
Comment (TPGPL) [inline wiki/Glossary/Snapping/en.md:7]:
```suggestion
The term **basic-snapped** is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) whose time between [active objects](/wiki/Glossary/Active_object) is at least two times longer than the time required to allow dashes or hyperdashes, respectively. For example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic-snapped hyperdash.
```

File: wiki/Glossary/Snapping/en.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Higher-snapped
+
+The term *higher-snapped* is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) that doesn't meet the requirement to be a basic-snapped one (i.e. the time between the objects is less than the threshold to be classified as basic).
```
Comment (TPGPL) [inline wiki/Glossary/Snapping/en.md:11]:
```suggestion
The term **higher-snapped** is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) that doesn't meet the requirement to be a basic-snapped one (i.e. the time between the objects is shorter than the threshold to be classified as basic).
```

File: wiki/Glossary/Trigger_distance/en.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Trigger Distance
```
Comment (TPGPL) [inline wiki/Glossary/Trigger_distance/en.md:1]:
```suggestion
# Trigger distance
```

File: wiki/Glossary/Trigger_distance/en.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
...
+
+*Trigger distance* is the term used in [osu!catch](/wiki/Game_mode/osu!catch) for the minimal spacing needed to generate a [hyperfruit](/wiki/Hit_object/Hyperfruit) on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop).
```
Comment (TPGPL) [inline wiki/Glossary/Trigger_distance/en.md:3]:
```suggestion
**Trigger distance** is a term used in [osu!catch](/wiki/Game_mode/osu!catch) for the minimal spacing needed to generate a [hyperfruit](/wiki/Hit_object/Hyperfruit) on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop).
```

File: wiki/Glossary/Walk/en.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Walk
+
+A *walk* is the term used in [osu!catch](/wiki/Game_mode/osu!catch) for the movement done between two objects when the player can catch both objects without pressing the dash key.
```
Comment (TPGPL) [inline wiki/Glossary/Walk/en.md:7]:
```suggestion
The **walk** is a term used in [osu!catch](/wiki/Game_mode/osu!catch) for the movement done between two objects when a player can catch both objects without pressing the dash key.
```

File: wiki/Hit_object/Hyperfruit/en.md [hunk 10]
```diff
@@ -4,7 +4,7 @@ stub: true
 
 # Hyperfruit
 
-A *hyperfruit* is an effect that can be generated on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop). This effect will be generated when the next fruit or drop can't be caught with normal dashing. The movement that is created when catching the hyperfruit is called a hyperdash. This effect will also generate when the last [banana](/wiki/Hit_object/Banana) of a [spinner](/wiki/Hit_object/Spinner) can't be caught with a dash.
+A *hyperfruit* is an effect that can be generated on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop). This effect will be generated when the next fruit or drop can't be caught with normal dashing. The movement that is created when catching the hyperfruit is called a [hyperdash](/wiki/Glossary/Hyperdash). This effect will also generate when the last [banana](/wiki/Hit_object/Banana) of a [spinner](/wiki/Hit_object/Spinner) can't be caught with a dash.
```
Comment (TPGPL) [inline wiki/Hit_object/Hyperfruit/en.md:7]:
```suggestion
The **hyperfruit** is an effect that can be generated on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop) when the next fruit or drop can't be caught with normal dashing. The movement that is created after catching the hyperfruit is called a [hyperdash](/wiki/Glossary/Hyperdash). This effect will also generate when the last [banana](/wiki/Hit_object/Banana) of a [spinner](/wiki/Hit_object/Spinner) can't be caught with a dash.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
my note about `aniflow` and `snapping` not being osu!catch-specific still stands -- please move it to a better home, which, in my opinion, is `/wiki/Disambiguation/`

**Comment by Darius-Wattimena** [comment]
The snapping thing is something used only in osu!catch as it is a term used to identify strong and weak snapping movements, making it unrelated to object snapping (e.g. 1/2 1/4). Anti-flow on the other hand does make sense to move to this folder. 

Anyway will make the needed changes in a couple of days once I wrapped up my thesis.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**